examples/19-EnsembleModel-examples.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<p>Ensembling combines the forecasts of several "weak" models to obtain a more robust and accurate model.</p><p> </p><p>All of Darts' ensembling models rely on the <strong>stacking technique</strong> (<a href="https://machinelearningmastery.com/essence-of-stacking-ensembles-for-machine-learning/" target="_blank">reference</a>). They provide the same functionalities as the other forecasting models. Depending on the ensembled models, they can:</p><ul><li>levarage covariates</li><li>be trained on multiple-series</li><li>predict multivariate targets</li><li>generate probabilistic forecasts</li><li>and more</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/1815/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>